v3/src/data-interactive/handlers/component-handler.ts

```diff
@@ -63,10 +66,10 @@ export const diComponentHandler: DIHandler = {
         (_type, createOrShowOptions) => document.content?.createOrShowTile(_type, createOrShowOptions)
       const _createOrShow = createOrShow ?? defaultCreateOrShow
       const _options = options ?? {}
+      const title = _title ?? name
+      const newTileOptions = { cannotClose, content, ...dimensions, name, title, ..._options }
+      const tile = _createOrShow(kComponentTypeV2ToV3Map[type], newTileOptions)
```

Comment:
The model changes should be made within the call to `applyModelChange()`, so these lines should move back where they were. You can use the same approach to getting access to the tile that is used elsewhere:
```
    let tile: Maybe<ITileModel>
```
and then set it.